common-content/en/module/databases/databases-with-sheets/_index.md

```diff
@@ -0,0 +1,83 @@
+
+Have a think about what else you do on your computer and phone, what information is there, where do you think it's stored?
+
+### How do they work?
+
+Most databases arrange data into columns and rows.
+
+- A column is used to store various properties about the data. In the example of your phones contact list, this would be name, mobile number, home phone number etc.
+
+- A row is then a record of this information. So an entry into your phone book, it will have a name e.g. Tom, and a phone number.
+
+| First Name    | Surname | Mobile Number |
+| ------------- | ------- | ------------- |
+| Tom           | Jones   | +447700000    |
+| Ali           | Smith   | +327700000    |
+
+Tom and Ali's details here are records in the database, where the information is stored in rows, and the properties of what we're storing is the columns.
+
+If you've used tools such as Excel or Google sheets, you will have seen this sort of structure before. We'll take a closer look at these in the backlog exercises to do this sprint.
+
+############ The below will be put into issues, just putting it in a PR to get any feedback :) #############
+
+# Backlog 1 - Learn about Google Sheets
+
+## Part 1 (Optional - Depending on knowledge of spreadsheets already.)
+
+Start by making sure we're familiar with Google sheets. The following tutorials will walk you through how to use a spreadsheet, format data, use functions, filters, and create charts using data.
+
+- [Build, organize, and format a spreadsheet using Google Sheets.](https://applieddigitalskills.withgoogle.com/c/college-and-continuing-education-uk/en-uk/google-workspace-sheets-part-1/google-workspace-sheets-part-1/important-lesson-update.html)
+
+- [Evaluate and interpret data with different spreadsheet functions, filters, and charts using Google Sheets.](https://applieddigitalskills.withgoogle.com/c/college-and-continuing-education-uk/en-uk/g-suite-certification-sheets-part-2/overview.html)
+
+- [Use advanced functions and conditional formatting in Google Sheets.](https://applieddigitalskills.withgoogle.com/c/college-and-continuing-education-uk/en-uk/g-suite-certification-sheets-part-3/overview.html)
+
+### Part 2 - Learn how to query data in Google Sheets using SQL
+
+Now you've got a good understanding of Google Sheets. We can start to use SQL, a powerful tool you will regularly come across when working with databases.
+
+What the video below, copying the dataset provided in the video and follow along with the queries as you go.
+
+- [SQL in Google Sheets](https://www.youtube.com/watch?v=dTNamL30sDg)
+- [Here is a link to the data](https://docs.google.com/spreadsheets/d/1zPeHxhKkYmdiqXNhMwhfC1A0o1wI0zNiD5Gsks2SYoc/template/preview)
+
+Do up to at the step `least GROUP BY clause and aggregation`. We can skip PIVOT, LIMIT and OFFSET for now, you wont need these for the next piece of coursework but feel free to look at them if you're curious and not short on time.
+
+# Backlog 2 - SQL Problems using Google Sheets
```

Comment:
BTW, there's a template for assigning coursework that should be stored on each repo. It's in the .github folder and should pop up when you hit create new issue

If you'd like to update the template, please do, but try to use a template so assignments are clearly structured. The current template is designed from long experience! 😉